external-docs/docs/guides/standard-shape.md

```diff
@@ -0,0 +1,70 @@
+The standard shape extends the [common structure](./shape-data.md#common-structure) by requiring that **each row** has a _measures_ column and a _units_ column; these columns define the measure and unit (of measure) for each row.
```

Comment:
Each markdown document should have a top-level title (e.g. `# This is the page title`).

Could you also make sure to provide an introduction summarising what the document contains and what knowledge it assumes, e.g.

> This page describes how to build and configure a standard shape data cube. For more information about the shapes of data supported by csvcubed, see the page on [shaping your data](todo: link to parent).

It may also be useful to point out that this doc expects a basic understanding of writing a qube-config.json file, and provide a link to the (currently bad) documentation. 

And can you do something similar in the pivoted shape document?

Reply:
Done